tests/test_starfileio.py

```diff
@@ -35,8 +36,12 @@ def grouper(iterable, n, fillvalue=None):
 
 class StarFileTestCase(TestCase):
     def setUp(self):
-        with importlib_resources.path(tests.saved_test_data, "sample.star") as path:
-            self.starfile = StarFile(path)
+        with importlib_resources.path(
+            tests.saved_test_data, "sample_data_model.star"
+        ) as path:
+            # explicitly reset blocks variable, ensures identical object is tested
```

Comment:
I'm confused by this. Does this have to do something with object persistence within the StarFile object?

Reply:
it had to do with a problem we had with the constructor of `StarFile`. It has been fixed by setting `blocks=None` by default. This line has been reverted to constructing from filepath, and the comment was removed to avoid confusion